docs/visual-builder/index.md

```diff
@@ -0,0 +1,117 @@
+# Visual Builder for agents
+
+<div class="language-support-tag">
+  <span class="lst-supported">Supported in ADK</span><span class="lst-python">Python v1.18.0</span><span class="lst-preview">Experimental</span>
+</div>
+
+The ADK Visual Builder is a web-based tool that provides a visual workflow
+design environment for creating and managing ADK agents. It allows you to
+design, build, and test your agents in a beginner-friendly graphical interface,
+and includes an AI-powered assistant to help you build agents.
+
+![Visual Agent Builder](../assets/visual-builder.png)
+
+!!! example "Experimental"
+    The Visual Builder feature is an experimental release. We welcome your
+    [feedback](https://github.com/google/adk-python/issues/new?template=feature_request.md)!
+
+## Get started
+
+The Visual Builder interface is part of the ADK Web tool user interface.
+Make sure you have ADK library
+[installed](/adk-docs/get-started/installation/#python)
+and then run the ADK Web user interface.
+
+```console
+adk web --port 8000
```

Comment:
Port 8000 is the default - should we remove the flag here?

Reply:
I put the --port flag on there because that's usually the thing that causes a failure to load (port already in use). So including the port setting it makes easy to adjust (and you know what port it's grabbing).